pkg/controllers/user/helm/common.go

```diff
@@ -21,9 +21,10 @@ import (
 )
 
 const (
-	helmName    = "rancher-helm"
 	appLabel    = "io.cattle.field/appId"
 	failedLabel = "io.cattle.field/failed-revision"
+	helmNameV2  = "rancher-helm"
+	helmNameV3  = "helm_v3"
```

Comment:
If you export these from common/common.go then you don't need to duplicate them here